app/models/support_user.rb

```diff
@@ -0,0 +1,7 @@
+class SupportUser < ActiveRecord::Base
```

Comment:
Open question - there's an interesting inconsistency between provider and support: 

- support uses both `DfESignInUser` and `SupportUser` (an AR model)
- the provider interface only uses `ProviderUser`, which is a sublass of `DfESignInUser`

How would we make those two things behave the same?

Reply:
It is inconsistent right now. Duncan and I were discussing yesterday. He wants to put 'the list' of authorised provider users into the database too (having a .yml file means redeploying to make changes). He was keen to get started on this yesterday but was waiting on my PR. So it's going to change some more and (I think) become a lot more consistent.

Right now `DfESignInUser` represents an authenticated user whereas `SupportUser` represents an authorised user. On the Provider side it's a bit different because the authorisation check is done at the point the user attempts to query for applications rather than immediately after authentication.